### ISSUE TITLE
Use correct fileKey for storage and lookup

### ISSUE THREAD

**Issue by aldeed** [CollectionFS/Meteor-CollectionFS#252]
@raix, in cfs-storage-adapter FS.Transform.prototype.createReadStream and FS.Transform.prototype.createWriteStream, it generates the fileKey using the SA `fileKey` method. I think it would be better for both read and write streams to use the key we have stored in `fileObj.getCopyInfo(storeName).key`, if set. This ensures that it's the identical key returned from the initial storage. Otherwise, if any SA changes its `fileKey` method, we can no longer access anything already stored.

In other words, api.fileKey should be used for new file writes only.

I can make this change if you want, but I wanted to make sure you agree.
